var/spack/repos/builtin/packages/flecsph/package.py

```diff
@@ -15,24 +14,20 @@ class Flecsph(CMakePackage):
     includes support for gravitational forces using the fast multipole method
     (FMM)."""
 
-    homepage = "http://flecsi.lanl.com"
+    homepage = "http://flecsi.org"
     git = "https://github.com/laristra/flecsph.git"
 
+    maintainers("JulienLoiseau")
     version("master", branch="master", submodules=True, preferred=True)
 
-    variant("test", default=True, description="Adding tests")
+    variant("test", default=False, description="Enable unit testing")
```

Comment:
This shouldn't be a variant (it doesn't result in different binaries being built).

Reply:
I removed the variant.